aleph-client/src/lib.rs

```diff
@@ -301,3 +303,49 @@ pub fn get_storage_key(pallet: &str, call: &str) -> String {
     let storage_key = StorageKey(bytes.into());
     hex::encode(storage_key.0)
 }
+
+pub type RewardPoint = u32;
+
+/// Helper to decode reward points for an era without the need to fill in a generic parameter.
+/// Reward points of an era. Used to split era total payout between validators.
+///
+/// This points will be used to reward validators and their respective nominators.
+#[derive(Clone, Decode, Default)]
+pub struct EraRewardPoints {
```

Comment:
Thanks to @kostekIV contribution to Substrate (https://github.com/paritytech/substrate/pull/11422), the fields of this struct are now public, so we can use struct directly from staking pallet